src/EditorFeatures/Text/Extensions.SnapshotSourceText.cs

```diff
@@ -213,7 +203,7 @@ public override SourceText WithChanges(IEnumerable<TextChange> changes)
                 }
 
                 // check whether we can use text buffer factory
-                var factory = TextBufferFactory;
+                var factory = _textBufferCloneServiceOpt;
```

Comment:
Nit: now that that's readonly, we don't have to worry about capturing to a local which was previously required. (But hey, it's shorter than inlining, so...)

Reply:
I would likely change it (at least rename) if the code was more complex, but the method seems straightforward enough to keep.